contracts/JetStakingV2.sol

```diff
@@ -426,11 +434,58 @@ contract JetStakingV1 is AdminControlled {
         );
     }
 
+    /// @dev extend Aurora stream reward schedule. It can be only called by the default
+    /// contract admin. It can accept extending the schedule if the new schedule is
+    /// not overlapping with the current schedule. Also, it requires to pause the contract before calling it.
+    /// @param scheduleTimes timestamp denoting the start of each scheduled interval. Last element is the end of the stream.
+    /// @param scheduleRewards remaining rewards to be delivered at the beginning of each scheduled interval. Last element is always zero.
+    function extendAuroraStreamSchedule(
+        uint256[] memory scheduleTimes,
+        uint256[] memory scheduleRewards
+    ) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
+        require(paused != 0, "REQUIRE_PAUSE");
+        Stream storage stream = streams[0];
+        // requires ending the current stream in order to extend it.
+        // This will avoid mixing old and new rewards.
+        require(
+            scheduleTimes[0] >
+                stream.schedule.time[stream.schedule.time.length - 1],
+            "INVALID_SCHEDULE_START_TIME"
```

Comment:
```suggestion
            scheduleTimes[0] ==
                stream.schedule.time[stream.schedule.time.length - 1],
            "INVALID_SCHEDULE_START_TIME"
```
Since rewards have already been released based on the end index in [schedule](https://github.com/aurora-is-near/aurora-staking-contracts/blob/4d258cb4c3e2f122ef1e6f0a55a824e72030ef7c/contracts/JetStakingV1.sol#L981), shouldn't this extended schedule start where the previous ended.

Reply:
Updating the tests 👍